### DISCUSSION TITLE
Using performance testing tools in the Pimcore admin

### DISCUSSION THREAD

**Discussion by vriveramgmglobant** [pimcore/pimcore#16045] [Q&A / Support]
Hi Pimcore community

Is it possible to use performance testing tools in the admin panel? I'm using k6 at the moment, I wrote a simple script to log in and then open an object. I generated the CSRF token and sent it in the login request, but it keeps redirecting me to the login page. So now I'm wondering if it is possible to use any performance test tool in the admin. I'm not developing anything for a front page, my project is admin-based.

Thanks in advance for any advice anyone can give.
Regards

**Comment by dpfaffenbauer** [comment]
I did some automated testing with behat and panther for the backend. so it is possible for sure ;)
